Misc/NEWS.d/next/Library/2024-10-16-04-50-53.gh-issue-125542.vZJ-Ns.rst

```diff
@@ -1 +1,2 @@
-Deprecate passing prefix_chars_ parameter to :meth:`ArgumentParser.add_argument_group`.
+Deprecate passing keyword-only ``prefix_chars`` argument to
```

Comment:
If references do no work, let use the style usually used for function parameters.
```suggestion
Deprecate passing keyword-only *prefix_chars* argument to
```